clang/lib/Sema/SemaDecl.cpp

```diff
@@ -357,10 +357,13 @@ ParsedType Sema::getTypeName(const IdentifierInfo &II, SourceLocation NameLoc,
       return nullptr;
   }
 
-  // FIXME: LookupNestedNameSpecifierName isn't the right kind of
-  // lookup for class-names.
-  LookupNameKind Kind = isClassName ? LookupNestedNameSpecifierName :
-                                      LookupOrdinaryName;
+  // In case we know that the identifier is a class name, we know that it is
```

Comment:
```suggestion
  // In the case where we know that the identifier is a class name, we know that it is
```

Reply:
Sorry for the bad wording/sentence structure. English is my second language :).

Reply:
Absolutely not a problem, happens all the time.  Thank you very much for the patch!  Let us know if you need someone to commit this for you, or if you already have merge rights.

Reply:
Thank you. I have merge rights. I will maybe wait a little bit, then I will squash and merge the change.